src/components/Tweet.js

```diff
@@ -1,34 +1,25 @@
-function Tweet() {
+import React from "react";
+import Actions from "./Actions";
+import ProfileImage from "./ProfileImage";
+import User from "./User";
+import Timestamp from "./Timestamp";
+import Message from "./Message";
+
+function Tweet(props) {
   return (
     <div className="tweet">
-      <img
-        src="https://i.imgur.com/9yw1Fyw.jpg"
-        className="profile"
-        alt="profile"
-      />
+      <ProfileImage image={props.tweet.User.image} />
```

Comment:
here you have a typo : you write "user" with uppercase 
it should be lowercase : 
image={props.tweet.user.image}